### PR TITLE
FROST audit announcement

### PR DESCRIPTION
**Pull request by chelseakomlo**

### CODE REVIEW

**Review by upbqdn — CHANGES_REQUESTED** [review]
I noticed two typos:
- We have finished implemented their...
- ... that major security issues... [missing "no"?]

File: _posts/blog/2021-03-29-frost-audit.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+layout: post
+title: "The Zcash Foundation has Finished an Audit of FROST"
+excerpt: "FROST Implementation Audit"
+categories: blog
+tags: [frost]
+Date: 2021-03-29
+author: zfteam
+---
+
+The Zcash Foundation is pleased to announce our completion of an audit for our first implementation [FROST implementation](https://github.com/ZcashFoundation/redjubjub/blob/main/src/frost.rs). This implementation is a first step and does not yet include support for re-randomziation nor the DKG, but these will be features that we will add next (and will complete separate audits for each).
```
Comment (dconnolly) [inline _posts/blog/2021-03-29-frost-audit.md:11]:
```suggestion
The Zcash Foundation is pleased to announce our completion of an audit for our first [FROST implementation](https://github.com/ZcashFoundation/redjubjub/blob/main/src/frost.rs). This implementation is a first step and does not yet include support for re-randomziation nor the DKG, but these will be features that we will add next (and will complete separate audits for each).
```

File: _posts/blog/2021-03-29-frost-audit.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+For this audit, we worked with JP Aumasson and Adrian Hamelink from the [Taurus Group](https://www.taurushq.com) and Omer Shlomovits from [ZenGo](https://www.zengo.com/), who have deep experience [evaluating the security](https://eprint.iacr.org/2020/1052) of multi-party signature schemes. We have finished implementing their recommended security measures, and are in the process of fixing
```
Comment (veorq) [inline _posts/blog/2021-03-29-frost-audit.md:13]: Please change to "from [Taurus](https://www.taurushq.com)", we just renamed the company to get rid of the "Group"  :)

**Review by upbqdn — APPROVED** [review]

File: _posts/blog/2021-03-29-frost-audit.md (hunk 2 above)
Comment (dconnolly) [inline _posts/blog/2021-03-29-frost-audit.md:13]:
```suggestion
For this audit, we worked with JP Aumasson and Adrian Hamelink from [Taurus](https://www.taurushq.com) and Omer Shlomovits from [ZenGo](https://www.zengo.com/), who have deep experience [evaluating the security](https://eprint.iacr.org/2020/1052) of multi-party signature schemes. We have finished implementing their recommended security measures, and are in the process of fixing
```

File: _posts/blog/2021-03-29-frost-audit.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+ up recommended usability recommendations now. We are happy to say that no major security issues were found in our implementation.
+
+The full audit findings can be found [here](). We will be releasing a first stable version of this implementation in the coming weeks. As said, we are currently implementing features for full compatibility with the Zcash protocol by finalizing the design of re-randomization in a threshold setting, and will also implement the DKG as specified in the [FROST paper](https://eprint.iacr.org/2020/852). We’ll keep the community updated on our progress as we go along.
```
Comment (teor2345) [inline _posts/blog/2021-03-29-frost-audit.md:16]: TODO
> [here]()

File: _posts/blog/2021-03-29-frost-audit.md (hunk 2 above)
Comment (chelseakomlo) [reply _posts/blog/2021-03-29-frost-audit.md:13]: Sure thing.

**Pull request merged**